File: C++/Iniciante/1008.cpp
#include <stdio.h>
#include <iostream>

int main() {
  
  int n, n_horas;
  double valor_hora, salario;

  std::cin >> n;
  std::cin >> n_horas;
  std::cin >> valor_hora;

  salario = n_horas * valor_hora;
  
  printf("NUMBER = %d\nSALARY = U$ %.2lf\n", n, salario);
  
}

File: R/Iniciante/1008.r
input <- file('stdin', 'r')

n <- as.integer(readLines(input, n = 1))
n_horas <- as.integer(readLines(input, n = 1))
valor_hora <- as.double(readLines(input, n = 1))

salario = n_horas * valor_hora

cat(sprintf("NUMBER = %d\nSALARY = U$ %.2f\n", n, salario))
